pkg/innerring/innerring.go

```diff
@@ -425,7 +425,7 @@ func New(ctx context.Context, log *zap.Logger, cfg *viper.Viper, errChan chan<-
 	}
 
 	if !server.sideNotaryConfig.disabled {
-		// enable notary support in the side client
+		// enable notary support in the sideclient
```

Comment:
i think one of the:
1. keep as it was done;
2. `sidechain client`.

Reply:
Oh, thanks, my typo